src/main/java/org/folio/service/file/upload/FileUploadServiceImpl.java

```diff
@@ -59,4 +90,17 @@ public FileUploadServiceImpl(@Autowired FileStorage fileStorage, @Autowired File
       .compose(fileDefinition -> fileDefinitionService.update(fileDefinition.withStatus(ERROR), tenantId));
   }
 
+
+  private Future<FileDefinition> updateFileDefinitionWithJobExeution(JobExecution jobExecution, FileDefinition fileDefinition, String tenantId) {
+    return jobExecutionService.save(jobExecution, tenantId)
+      .compose(savedJob -> fileDefinitionService.update(fileDefinition.withJobExecutionId(savedJob.getId()), tenantId));
+  }
+
+  private JobExecution getJobExecutionWithProgress(int total) {
```

Comment:
do we really need this method with only one line of code? is it used twice somewhere?

Reply:
usage of such method make code more readable in the place where it is called